### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by asnowwolf**
Add `markup-inline-loader`

### CODE REVIEW

File: content/loaders/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
 * `markdown-loader` Compiles Markdown to HTML
 * `posthtml-loader` Loads and transforms a HTML file using [PostHTML](https://github.com/posthtml/posthtml)
 * `handlebars-loader` Compiles Handlebars to HTML
-
+* [`markup-inline-loader`](https://github.com/asnowwolf/markup-inline-loader) Inline SVG/MathML files to HTML. It’s very useful when applying icon font or applying CSS animation to SVG.
```
Comment (bebraw) [inline content/loaders/index.md]: The linter chokes on *very*. Can you drop the word? Good to go after that.

**Comment by asnowwolf** [comment]
Done.
Sorry for my poor English ^_^ .

**Comment by simon04** [comment]
@asnowwolf, make sure to submit a PR for https://github.com/webpack-contrib/awesome-webpack#loaders as well.

**Comment by asnowwolf** [comment]
@simon04 Done, Thank you!

**Pull request merged**